### PR TITLE
socrata sync

### LINKED ISSUE #188: Socrata Sync
> as for the KB, I would just say to wait until Monday so we're certain that it's working

### PR DESCRIPTION
**Pull request by aviv**
closes #188

### CODE REVIEW

File: contents/articles/screendoor/integrations/socrata.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Screendoor makes it easy to passively sync responses to a Socrata dataset.
+
+## Connecting to Socrata
+
+To sync with Socrata, you'll first need to make sure you have a Socrata account. Then, click "Socrata sync" from the Settings page for your project.
+
+![connect to socrata](../images/socrata_connect.png)
+
+Fill in the domain of your Socrata portal and click "Connect your Socrata Account." You'll be redirected to the Socrata sign-in page (unless you're already signed in). Sign in to Socrata and click the "Allow" button to go back to Screendoor.
```
Comment (ajb) [inline contents/articles/screendoor/integrations/socrata.md]: > the domain of your Socrata portal

Might be confusing for some folks.

Comment (aviv) [inline contents/articles/screendoor/integrations/socrata.md]: yah i'm actually a bit confused about how this works re: socrata's custom domains and such

File: contents/articles/screendoor/integrations/socrata.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Choosing responses to sync
+
+Next, choose which responses you'd like to sync to Socrata: Click the "Choose responses" button and enter a search query and/or add custom filters. To sync all responses, just leave these blank.
```
Comment (ajb) [inline contents/articles/screendoor/integrations/socrata.md]: I feel like this step has the biggest potential to confuse people. Is there anything else we can say here?

**Comment by ajb** [comment]
:+1:

This also relates to https://github.com/dobtco/blog-v2/pull/33.

Not sure who's gonna take on the screencast, but it would be cool if we could somehow cross-link these pages, or embed the screencast here, too.

File: contents/articles/screendoor/integrations/socrata.md (hunk 1 above)
Comment (ajb) [inline contents/articles/screendoor/integrations/socrata.md]: Maybe just define that "domain" == web address?

On Thu, Apr 9, 2015 at 4:45 PM, Aviv Nitsan <notifications@github.com>
wrote:

> In contents/articles/screendoor/integrations/socrata.md
> <https://github.com/dobtco/knowledge_base/pull/190#discussion_r28100854>:
>
> > +---
> > +title: Socrata data sync
> > +order: 1
> > +app_pages:
> > +  - projectadmin-socratasyncs-show
> > +---
> > +
> > +Screendoor makes it easy to passively sync responses to a Socrata dataset.
> > +
> > +## Connecting to Socrata
> > +
> > +To sync with Socrata, you'll first need to make sure you have a Socrata account. Then, click "Socrata sync" from the Settings page for your project.
> > +
> > +connect to socrata
> > +
> > +Fill in the domain of your Socrata portal and click "Connect your Socrata Account." You'll be redirected to the Socrata sign-in page (unless you're already signed in). Sign in to Socrata and click the "Allow" button to go back to Screendoor.
>
> yah i'm actually a bit confused about how this works re: socrata's custom
> domains and such
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dobtco/knowledge_base/pull/190/files#r28100854>.
>



-- 
Adam Becker
(951) 9-BECKER
@AdamJacobBecker

**Comment by kariannemah** [comment]
per my call with @jkgoldst an hour ago:
the tentative plan for the screencast is that he will write the script and I will narrate 
:microphone::microphone::microphone:

**Pull request merged**